account_multicurrency_revaluation_rate_type/model/res_company.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    revaluation_rate_type = fields.Selection(
+        string="Rate type",
```

Comment:
```suggestion
        string="Revaluation rate type",
```
string arg to update.